packages/mdc-shape/_variables.scss

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+$mdc-shape-shadow-padding: 56px;
+$mdc-shape-shadow-negative-padding: -1 * $mdc-shape-shadow-padding;
```

Comment:
Is there a motive behind making this a variable rather than simply doing -1 * mdc-shape-shadow-padding in the 2 cases where this is needed? The presence of a separate variable makes me think there'd be more to it than simply negating the other one.

Reply:
uh nope, this was silly. I removed it.